pyproject.toml

```diff
@@ -50,9 +52,6 @@ ignore_patterns = [
 line_length = 160
 
 [tool.mypy]
-strict = true
```

Comment:
@mamo3gr 

need to  remove `strict` ?

Reply:
@yokomotod 
Should it be redirected to @maronuu ?

Reply:
kusa

Reply:
😅 😅 😅

Reply:
Putting that aside, I srtongly disaggree with removing the strict setting.